homeassistant/components/zabbix/__init__.py

```diff
@@ -159,7 +163,25 @@ def event_to_metrics(
             )
             metrics.append(metric)
 
-        string_keys.update(strings)
```

Comment:
by moving this into the `if` block, and since `include_strings` defaults to `False`, isn't this a breaking change?